### PR TITLE
{data,lib,math}[GCC/12.3.0,GCCcore/12.3.0,foss/2023a] PEcAn v1.7.2.9000, Redland v1.0.17, unixODBC v2.3.12 w/ R 4.3.2

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PEcAn/PEcAn-1.7.2.9000-foss-2023a-R-4.3.2.eb [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    ('storr', '1.2.5', {
+        'checksums': ['4224c3991d9c043a45ce530d0698d7f2cdca231b26fe31b45e0db865026e5f63'],
+    }),
+    ('thor', '1.1.5', {
+        'checksums': ['99802d8c8471ce351403f8428c1efab48ccf20f359fbd368db84f25ef8f7ecae'],
+    }),
+    ('duckdb', '1.0.0-2', {
+        'checksums': ['3735d51f22d62a7411aa514142cd569ce28bdaec4eb58d5f0893a476b9ccbf01'],
+        # fixes the `prefix_back_ was not declared` error, see https://github.com/duckdb/duckdb/pull/12344
+        'preinstallopts': "sed -i '%s' src/duckdb/third_party/re2/re2/prog.cc && " % local_sed_patterns_duckdb,
```
Comment (boegel) [inline easybuild/easyconfigs/p/PEcAn/PEcAn-1.7.2.9000-foss-2023a-R-4.3.2.eb:117]: @PetrKralCZ Why not use a patch file here? (see https://patch-diff.githubusercontent.com/raw/duckdb/duckdb/pull/12344.diff)

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/p/PEcAn/PEcAn-1.7.2.9000-foss-2023a-R-4.3.2.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/PEcAn/PEcAn-1.7.2.9000-foss-2023a-R-4.3.2.eb]: fixed in bed9e08

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/96e69ec74e4e5cebfe5cd9d74f69f0de for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**